conda_forge_tick/auto_tick.xsh

```diff
@@ -88,20 +127,23 @@ def main():
                 rerender = (gx.nodes[node].get('smithy_version') != smithy_version or
                             gx.nodes[node].get('pinning_version') != pinning_version or
                             migrator.rerender)
-                run(attrs=attrs, migrator=migrator, gh=gh, rerender=rerender, protocol='https',
-                    hash_type=attrs.get('hash_type', 'sha256'))
-                # TODO: capture pinning here too!
-                gx.nodes[node].update({'PRed': attrs['new_version'],
-                                       'smithy_version': smithy_version,
+                migrator_uid, pr_json = run(attrs=attrs, migrator=migrator, gh=gh,
+                                    rerender=rerender, protocol='https',
+                                    hash_type=attrs.get('hash_type', 'sha256'))
+
+                converted_hash = migrator_uid
```

Comment:
I think this line can be removed and every instance of  `converted_hash` changed to `migrator_uid`